next-frontend/package.json

```diff
@@ -3,18 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev --turbopack",
```

Comment:
I knew we would have to disable the experimental turbopack support sooner or later...